plugins/Live/Live.php

```diff
@@ -68,11 +69,11 @@ public static function checkIsVisitorLogEnabled($idSite = null): void
         }
 
         if (empty($idSite)) {
```

Comment:
The changes below enables the methods to check if visitlog and profile are enabled to handle `idSite=all`. If any site has it disabled it will then return false.